packages/template/package.json

```diff
@@ -25,7 +25,7 @@
     "@types/node": "^20.14.6",
     "@types/react": "^18.3.3",
     "@unocss/reset": "^0.59.4",
-    "astro": "4.10.3",
+    "astro": "^4.10.3",
```

Comment:
This is required for `yarn`. Without this it would install two copied of `astro` and complain about type conflict in `astro.config.ts`.

Reply:
Oh good catch!